### PR TITLE
mt-update-ttl : use standard store, specify TTL's not tables, auto-create tables + misc

### PR DESCRIPTION
**Pull request by Dieterbe**

### CODE REVIEW

File: cmd/mt-update-ttl/main.go [hunk 1]
```diff
@@ -58,89 +36,73 @@ func init() {
...
+
+	flag.BoolVar(&verbose, "verbose", false, "show every record being processed")
+
 	flag.Usage = func() {
 		fmt.Fprintln(os.Stderr, "mt-update-ttl [flags] ttl table-in [table-out]")
 		fmt.Fprintln(os.Stderr)
 		fmt.Fprintln(os.Stderr, "Adjusts the data in Cassandra to use a new TTL value. The TTL is applied counting from the timestamp of the data")
 		fmt.Fprintln(os.Stderr, "If table-out not specified or same as table-in, will update in place. Otherwise will not touch input table and store results in table-out")
-		fmt.Fprintln(os.Stderr, "In that case, it is up to you to assure table-out exists before running this tool")
+		fmt.Fprintln(os.Stderr, "Unless you disable create-keyspace, table-out is created when necessary")
```
Comment (woodsaj) [inline cmd/mt-update-ttl/main.go]: This assumes that table-out is named using the internal naming convention of MT.  But there are no checks to ensure that is true.

Comment (Dieterbe) [reply cmd/mt-update-ttl/main.go]: i suppose we could check if the table generated for the requested TTL matches the name specified.
but at that point, we may as well get rid of the table-out argument and just always generate it automatically. mt-update-ttl then knows the right destination table without further user input, and even know when to do updates (if it's same as table-in) or copies (if it's different). does that sound good to you @woodsaj ?

Comment (woodsaj) [reply cmd/mt-update-ttl/main.go]: That seems reasonalbe, but you should then do the same for both the input and out tables. ie `old-ttl`, `new-ttl`

**Comment by Dieterbe** [comment]
@woodsaj PTAL

**Pull request merged**